assets/js/components/dashboard-sharing/ModuleRecoveryAlert/index.js

```diff
@@ -19,34 +19,23 @@
 /**
  * WordPress dependencies
  */
-import { Fragment, useCallback, useEffect, useState } from '@wordpress/element';
-import { sprintf, __ } from '@wordpress/i18n';
+import { useMemo } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
-import { useSelect, useDispatch } from 'googlesitekit-data';
-import { Checkbox, ProgressBar } from 'googlesitekit-components';
-import { CORE_SITE } from '../../../googlesitekit/datastore/site/constants';
+import { useSelect } from 'googlesitekit-data';
 import { CORE_MODULES } from '../../../googlesitekit/modules/datastore/constants';
-import { DAY_IN_SECONDS } from '../../../util';
-import BannerNotification from '../../notifications/BannerNotification';
-import Errors from './Errors';
-
-export default function ModuleRecoveryAlert() {
-	const [ checkboxes, setCheckboxes ] = useState( null );
-	const [ recoveringModules, setRecoveringModules ] = useState( false );
+import SimpleNotification from '../../../googlesitekit/notifications/components/layout/SimpleNotification';
+import Description from './Description';
+import Actions from './Actions';
 
+export default function ModuleRecoveryAlert( { id, Notification } ) {
 	const recoverableModules = useSelect( ( select ) =>
 		select( CORE_MODULES ).getRecoverableModules()
 	);
 
-	const documentationURL = useSelect( ( select ) => {
-		return select( CORE_SITE ).getDocumentationLinkURL(
-			'dashboard-sharing'
-		);
-	} );
-
 	const userAccessibleModules = useSelect( ( select ) => {
```

Comment:
I realize this is from before, but I noticed it in some of the new components. There is a mismatch of types which can easily happen looking at the names:

- `recoverableModules` (`{ [key: string]: Object }`) – subset of `getModules`
- `userAccessibleModules` (`string[]`) – list of module slugs: subset of `recoverableModules` that the user has access to

Both are `*Modules` but different types which is confusing. For a list of slugs a name like `*ModuleSlugs` would be much more clear 👍  Even once we start integrating TypeScript, we should still use the same level of care when naming things.

TL;DR If we have a use for user recoverable module objects, then we can remove the `slug` mapping at the end and move that outside to a new variable.